### ISSUE TITLE
Trakt

### ISSUE THREAD

**Issue by duramato** [scytherswings/Network-Status-Page#1]
Trakt.tv integration doesnt seem to be working

**Comment by scytherswings** [comment]
I believe trakt.tv might have issues since I added the ability to have multiple instances of each service. I'll see if I can get some testing in tonight after work.

**Comment by duramato** [comment]
@scytherswings  also how can i put the "Throughput" to work? the ping just displays "WAN1 Ping: Array ms"

**Comment by scytherswings** [comment]
I think the Throughput test requires you to integrate with pfsense. It should be pulling that information over ssh if that is working

**Comment by duramato** [comment]
@scytherswings  i do not have pfSense installed,could you point me to any good tuturial becouse i didnt found any good one

**Comment by scytherswings** [comment]
pfSense is a firewall, you would use it in place of your home router. It is an operating system that is based off of freeBSD and it can be run in a virtual machine or on a physical box. Perhaps I'll make a feature that can test the throughput without using pfSense since it is a very specific configuration.

**Comment by duramato** [comment]
i believe it would be better to be able to use it whithout pfSense ,but probably some day i will install it on a raspberry pi or something similiar :)

**Comment by scytherswings** [comment]
I agree, it makes it more flexible. That being said, you'd need to use an old desktop or something to get pfSense up and running, it needs a minimum of two network interfaces to work properly.

**Comment by duramato** [comment]
@scytherswings Coul this be related with the new Trakt v2 api?

**Comment by scytherswings** [comment]
I'll have to look into it, was it updated since d4kr22 released this project? If that's the case I haven't had time to look into the trakt api setup yet.

**Comment by duramato** [comment]
yes it was released earlier this year, and v1 support got droped  like a month or 2 ago afaik

**Comment by scytherswings** [comment]
This new API sucks and it's going to take a while for me to code a new interface. I'm looking into using a wrapper somebody else made, but the dependencies and crap are just spiraling out of control.

**Comment by scytherswings** [comment]
It looks like many things that this page was built on for trakt have changed on their end. I Probably could use some help figuring out how this should be redone. Until then, I'm going to disable Trakt until a solution is found.
